### ISSUE TITLE
RParser:  SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by apawlik** [stencila/dockter#26]
When compiling [R lesson for Software Carpentry](https://github.com/swcarpentry/r-novice-inflammation) I get an error:

```
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at CachedRequest.parseHeaders (/snapshot/dockter/node_modules/cached-request/lib/cached-request.js:112:15)
    at ReadStream.<anonymous> (/snapshot/dockter/node_modules/cached-request/lib/cached-request.js:194:37)
    at emitNone (events.js:111:20)
    at ReadStream.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at CachedRequest.parseHeaders (/snapshot/dockter/node_modules/cached-request/lib/cached-request.js:112:15)
    at ReadStream.<anonymous> (/snapshot/dockter/node_modules/cached-request/lib/cached-request.js:194:37)
    at emitNone (events.js:111:20)
    at ReadStream.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)

```

**Comment by nokome** [comment]
I think this is a non-fatal error emanating from `cached-request`. I think we can silence it safely. You should be able to `dockter compile ...` again and it will work.

**Comment by apawlik** [comment]
Umm, should I run it from `master` then? I am still testing it on the binaries.

**Comment by apawlik** [comment]
Fixed in 0.2.3 Thanks!
